great_expectations/compatibility/bigquery.py

```diff
@@ -21,6 +21,11 @@
 except (ImportError, AttributeError):
     GEOGRAPHY = SQLALCHEMY_BIGQUERY_NOT_IMPORTED
 
+try:
+    from sqlalchemy_bigquery import parse_url
+except (ImportError, AttributeError):
+    parse_url = SQLALCHEMY_BIGQUERY_NOT_IMPORTED
+
```

Comment:
This was the only module that was missing. It was added to `compatibility/bigquery.py`